app/e2e/common/pages/playwright-landing-page.ts

```diff
@@ -0,0 +1,158 @@
+// playwright-landing-page.ts
```

Comment:
I think this could be improved in a few ways:
1. I think this file is doing too much. 
2. I'm not sure the POM is the right model for us. I think we should tweak it.

On a philosophical level, we should ask "why are we testing?" It's to make sure we don't have any user facing breakages during a demo / event. 

What is a user facing breakage? Well, lets think about users for a moment, if a user needs to do step A, B, C, and D to complete something (say signing in), and step C is broken, then it doesn't matter that A, B, and D aren't broken. To the user, the web app doesn't work.

We also have limited bandwidth and every tests adds overhead. We shouldn't be testing everything. We can't.

Therefore, we should be strictly testing that user journeys don't break.  That is to say, what does it matter to the user if the login modal opens and closes, but logging in doesn't work? Why should we maintain separate tests for them?

With the previous API design I recommended, we could have something like the following, which would capture everything we care about for the user journey:

```ts
// Test for testing the landing page login
const I  = LandingPage({ page });

I.clickOnLogin();
I.see(LandingPage.loginFormModal);
I.fillInEmail("something@something.com");
I.fillInPassword("12345678");
I.clickLogin();
I.amLoggedIn();
```
Which encapsulates everything we need to test to make sure the user has a good experience. 

We can discuss the API more if you have questions, but those are my thoughts.

Reply:
Also, I want to add that not everything here needs to be done before merging. These were more so just my thoughts. With the way tests are currently being written, they're going to become overwhelming pretty quickly IMO.

Reply:
discussed/resolved in meeting